MaterialLib/MPL/VariableType.h

```diff
@@ -42,6 +42,7 @@ using Tensor = std::array<double, 9>;
 /// is missing, simply add it somewhere at the list, but above the last entry.
 enum class Variable : int
 {
+    concentration,
```

Comment:
```suggestion

```
Does not belong into this PR, I guess...

Reply:
Moved this commit into PR #2646